redaxo/src/addons/metainfo/lib/input/date.php

```diff
@@ -107,8 +107,9 @@ public function getYearSelect()
 
     public function getHtml()
     {
+        $thisYear = (int) date('Y');
         $yearSelect = clone $this->yearSelect;
-        $yearSelect->addOptions(range($this->startYear ?: 2005, $this->endYear ?: date('Y') + 10), true);
+        $yearSelect->addOptions(range($this->startYear ?: 2005, $this->endYear ?: $thisYear + 10), true);
```

Comment:
Könnte man auch über `idate()` lösen, die gibt ja direkt einen Integer zurück.

```suggestion
        $yearSelect->addOptions(range($this->startYear ?: 2005, $this->endYear ?: idate('Y') + 10), true);
```

Reply:
idate() kannte ich nicht... guter Tipp